jscomp/main/jsoo_refmt_main.ml

```diff
@@ -189,8 +189,8 @@ let make_compiler name impl prefix =
                         load_module cmi_path cmi_content (Js.to_string cmj_name) cmj_bytestring);
                   |]))
 
-let () = make_compiler "ocaml" Parse.implementation "[@@@bs.config{no_export}]\n#1 \"repl.ml\"\n"
-let () = make_compiler "reason" reason_parse "[@bs.config {no_export: no_export}];\n#1 \"repl.re\";\n"
```

Comment:
@bobzhang was the `repl.re` part needed? I couldn't find any references to it, so I removed it for simplicity, but better to be sure :)

Reply:
it gives you a file name in error message, we can keep it or change it to a more meaningful name